programmers/자릿수-더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(n)
+{
+    let count = 0
+    const numArr = n.toString().split("");
+    for (let i = 0; i < numArr.length; i += 1) {
+        count += Number(numArr[i]);
+    }
+    return count;
+}
```

Comment:
```suggestion
function solution(n) {
    const num = n.toString();
    let sum = 0
    
    for (let i = 0; i < num.length; i += 1) {
     sum += Number(num[i]);
    }
    return sum;
}
```

C스타일 for문 이외에 반복문을 사용할 수 있는것들이 있는데, 공부해주세요.
합을 구하는거니까  count보다는 sum이 좋을거 같아요!
split없이 답이 나오네요 ??